private/buf/bufconvert/bufconvert.go

```diff
@@ -46,6 +52,8 @@ var (
 	// sorted
 	messageEncodingFormats = []string{
 		formatBin,
+		formatBinProto,
+		formatTextProto,
 		formatJSON,
```

Comment:
```suggestion
		formatJSON,
		formatTextProto,
```
because of the `//sorted` comment